sentinel.go

```diff
@@ -200,10 +191,8 @@ func (s *Sentinel) newPool(addr string) *redis.Pool {
 // close connection pool to Sentinel.
 // Lock must be hold by caller.
 func (s *Sentinel) close() {
-	if s.pools != nil {
```

Comment:
We don't need this `nil` check because `len(nil) == 0`; the for loop will do nothing in that case.